x-pack/plugins/apm/public/components/shared/transaction_action_menu/custom_link_menu_section/custom_link_list.test.tsx

```diff
@@ -40,4 +53,15 @@ describe('CustomLinkList', () => {
     );
     expectTextsNotInDocument(component, ['foo', 'bar']);
   });
+
+  it('urlencode custom links with unsafe characters', () => {
+    const parsedLink = getParsedCustomLinkUrl(
+      customLinkWithTemplateVars,
+      transactionWithUnsafeCharacters
+    );
+
+    expect(parsedLink).toBe(
+      'http://elastic.co?service.name=foo%20bar/transactions?transactionName=#&#x2F;foo%20bar&#x2F;%2525c'
```

Comment:
This url doesn't look right. I think mustache is escaping the ampersand before you can, so it's double encoded. It should be possible to get back the original transaction name using

```
decodeURIComponent('#&#x2F;foo%20bar&#x2F;%2525c')
```

but it returns `#&#x2F;foo bar&#x2F;%25c` instead of `#/foo bar/%25c`